client/my-sites/comments/comment-list/index.jsx

```diff
@@ -95,24 +75,7 @@ export class CommentList extends Component {
 		changePage( page );
 	};
 
-	deleteCommentPermanently = ( commentId, postId ) => {
-		this.props.removeNotice( `comment-notice-${ commentId }` );
-
-		this.props.deleteComment( commentId, postId );
-	};
-
-	editComment = ( commentId, postId, commentData, undoCommentData, showNotice = true ) => {
-		this.props.editComment( commentId, postId, commentData );
-		if ( showNotice ) {
-			this.showEditNotice( commentId, postId, undoCommentData );
-		}
-	};
-
-	getComments = () => {
-		const comments = uniq( [ ...this.state.persistedComments, ...this.props.comments ] );
-
-		return orderBy( comments, null, this.state.sortOrder );
-	};
+	getComments = () => orderBy( this.props.comments, null, this.state.sortOrder );
```

Comment:
A note that we can try moving this to a memo'd selector later if this shows up in profiling.